### DISCUSSION TITLE
Cannot collect data in task 04

### DISCUSSION THREAD

**Discussion by xiedun2001** [deepmodeling/dpgen#471] [Q&A]
Dear developers,

I performed task 01 and task 04 in a Supercomputing Center. The task 01 finished correctly. It created 5 jobs in task 04. The dogen.log shows only 1 job finished.
![image](https://user-images.githubusercontent.com/68647398/126140804-7db4b53c-f9bb-4364-a60f-b1859acf45d3.png)
I checked the files in Supercomputing Center, found other 4 jobs also finished, but somehow the dpgen did not recognize it.
![image](https://user-images.githubusercontent.com/68647398/126140895-7dac371a-4969-45ae-9809-7e61774360ec.png)
It reported "status command squeue fails to execute" in nohup.out.
![image](https://user-images.githubusercontent.com/68647398/126140987-f0bbeadb-9c51-45ee-b4c8-33bd5f272742.png)
This is my machine.json. How to deal with this problem?
![image](https://user-images.githubusercontent.com/68647398/126141062-888454e2-6e9c-4085-85a9-e4c0dedc3eff.png)

**Comment by AnguseZhang** [comment]
1. Could you list the five folders in "work_path_dnn" via `ls *`  and see whether there are files called "xxxxx_tag_finished"?
2. Can DP-GEN automatically restart by `dpgen run param.json machine.json`?

**Reply by xiedun2001** [reply]
1.  All of the 4 folders have xxxxx_tag_finished file.
![image](https://user-images.githubusercontent.com/68647398/126253417-918f7344-58aa-4780-8be9-e9f5ee1b8c36.png)
![image](https://user-images.githubusercontent.com/68647398/126253432-ac2ef619-3667-4685-a1cf-dd4c6b2967ea.png)
![image](https://user-images.githubusercontent.com/68647398/126253445-e469520d-9824-4dc5-aa30-6927a9d4d496.png)
![image](https://user-images.githubusercontent.com/68647398/126253469-4b375ef0-5cee-41fa-a059-851b03b4805d.png)
2.  I tried `dpgen run param.json machine.json`. It shows:
![image](https://user-images.githubusercontent.com/68647398/126254725-2fc202d7-1245-40f0-bb74-2561af480373.png)
![image](https://user-images.githubusercontent.com/68647398/126254777-9df958fd-9034-4452-b297-762d62a2c3b9.png)

**Reply by AnguseZhang** [reply]
Could you try to execute command `squeue -o "%.18i %.2t" -j 30279` and see what is printed?

**Reply by xiedun2001** [reply]
The job is not in the sequence. 
![image](https://user-images.githubusercontent.com/68647398/126424125-2df82a34-c89b-4000-bec7-84620e82a0a8.png)

**Comment by felix5572** [comment]
Maybe because of slurm partition settings? 

How to query the job information in your cluster

**Reply by xiedun2001** [reply]
I use `parajobs` to  query the job which is running and in queue.

**Comment by xiedun2001** [comment]
I reduced the number of tasks from 2500 to 500, and reduce the md step from 3000 to 1000. The task 4 finished correctly. Dose the number and the size of result files in task 4 have a limit?

**Reply by felix5572** [reply]
Probably not. Maybe you could edit the source yourself, and print and debug the relating information yourself?

**Comment by njzjz** [comment]
I am wondering if the error was printed to `stdout` but not `stderr`. You can try `squeue -o "%.18i %.2t" -j 30279 1>/dev/null` and `squeue -o "%.18i %.2t" -j 30279 2>dev/null` to see if there is any difference.
